Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: push-down-test
+
+all: push-down-test
+
+push-down-test:
+	cd push-down-test && sh run-tests.sh
```

Comment:
`bash` but not `sh`

Reply:
I think `push-down-func-test` is better

Reply:
See: https://github.com/tikv/copr-test/pull/1#discussion_r335829782